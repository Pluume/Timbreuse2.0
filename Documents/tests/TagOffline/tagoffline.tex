\documentclass[10pt,a4paper,onecolumn]{article}
\usepackage[top=30pt,bottom=30pt,left=48pt,right=46pt]{geometry}
\usepackage{a4wide}
\usepackage{titling}
\setlength{\droptitle}{-5em}
\usepackage[version=3]{mhchem}
\usepackage{siunitx}
\usepackage{graphicx}
\usepackage{amsmath}
\usepackage[utf8]{inputenc}
\usepackage[english,francais]{babel}
\usepackage[backend=bibtex]{biblatex}
\usepackage[pdftex]{hyperref}
\setlength\parindent{0pt}
\usepackage{xcolor}
\hypersetup{colorlinks,urlcolor=blue}
\usepackage{fancyhdr}
\usepackage{lastpage}
\pagestyle{fancy}
\fancyhf{}
\chead{Francis Le Roy}
\lfoot{P1699/MCT}
\cfoot{\today}
\rfoot{Page \thepage \hspace{1pt} of \pageref{LastPage}}
% \bibliography{bib}
\renewcommand{\labelenumi}{\alph{enumi}.}



\title{Rapport de test \\ Timbrage hors connexion}

\author{Francis \textsc{Le Roy}}

\date{\today}

\begin{document}

\maketitle
\thispagestyle{fancy}

\begin{center}
\begin{tabular}{l r}
Date de réalisation : & 4 Avril, 2017 \\
Projet : & P1699 \\
\end{tabular}
\end{center}

\section{Introduction}
Il peut arriver que le réseau du CPNV tombe en panne pour une raison ou pour autre. Durant cette période de déconnexion, les élèves doivent pouvoir timbrer. Pour répondre à ce problème, la timbreuse doit être capable d'effectuer des actions de timbrage différées au moment de la reconnexion.
\section{Procédure de test}
Tout d'abord, timbrez sur une timbreuse à l'aide d'une carte d'un élève que l'on gardera durant toute l'expérience. Gardez en mémoire l'état actuel de l'élève que vous venez de faire timbrer. Éteignez le serveur et effectuez 3 timbrage avec la carte de l'élève sur la timbreuse. Rebranchez le serveur et attendez quelques secondes que les timbreuses se reconnectent. Lorsque le message affichant que la timbreuse est déconnectée s'en va, timbrez de nouveau. Si l'état est le même que lorsque vous aviez timbré pour la première fois, alors le test est un succès.
\section{Résultat obtenu}
En réalité, après la reconnexion, le serveur va traiter chacune des requêtes une par une. En revanche, il ne renvoie pas les informations relatives à l'élève à la timbreuse. Si plusieurs personnes ont timbré, ça n'aurait pas de sens de renvoyer les informations de la dernière personne traitée par le système.
\section{Conclusion}
Cette fonctionnalité est donc opérationnelle.\\
{\color{red}Attention ! Les élèves ne doivent pas timbrer sur des timbreuses différentes. Cela pourrait provoquer des résultats inattendus.}
% \printbibliography

\end{document}
