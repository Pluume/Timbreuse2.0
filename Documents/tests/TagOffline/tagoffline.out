\BOOKMARK [1][-]{section.1}{Introduction}{}% 1
\BOOKMARK [1][-]{section.2}{Proc\351dure de test}{}% 2
\BOOKMARK [1][-]{section.3}{R\351sultat obtenue}{}% 3
\BOOKMARK [1][-]{section.4}{Conclusion}{}% 4
